lib/msal-browser/src/interaction_client/SilentRefreshClient.ts

```diff
@@ -41,8 +41,8 @@ export class SilentRefreshClient extends StandardInteractionClient {
             .catch((e: AuthError) => {
                 if (e instanceof AuthError) {
                     (e as AuthError).setCorrelationId(this.correlationId);
+                    serverTelemetryManager.cacheFailedRequest(e);
```

Comment:
Was there a possibility `e` is not of the type `AuthError`? If NO, we can get rid of the `if` and if `YES`, we still need to cache it outside of the condition right?

Reply:
Good catch, thank you. `if (e instanceof AuthError)` is redundant here. Fixed.